jstests/core/profile_sampling.js

```diff
@@ -0,0 +1,48 @@
+// special db for running tests in parallel
+var stddb = db;
```

Comment:
Oooh, I see, this is so that `db` is rebound to the "test" database after this jstest completes. Instead of doing this, it's probably better to do something like

`var profileDB = db.getSisterDB("profile_sampling");`

Then you can use `profileDB` throughout the test instead of `db`. That way we're not updating the state of `db`, which other tests may depend on.